1_fetch/out/model_RMSEs.csv

```diff
@@ -0,0 +1,116 @@
+exper_n,exper_id,model_type,rmse
```

Comment:
This particular file is pretty small, so it matters less, but in general we steer away from committing data files, especially those for which we have scripted access to download them from elsewhere. GitHub has a storage limit that makes it suboptimal for storing data files. Can you remove this file from the repo temporarily, commit the removal, and add the file to `.gitignore` so that you're not tempted to commit it again later, and then replace the file in your local copy of the repo? More broadly, with pipelines I usually gitignore the whole `*/out/*` so that outputs of each function don't get committed by accident.